addons/fetchmail/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,7 @@
+---Models in module 'fetchmail'---
+---Fields in module 'fetchmail'---
+fetchmail    / fetchmail.server         / server_type (selection)       : NEW required, selection_keys: ['imap', 'local', 'pop'], req_default: function, hasdefault
+fetchmail    / fetchmail.server         / type (selection)              : DEL required, selection_keys: ['imap', 'local', 'pop'], req_default: function
+# Done in pre-migration script
```

Comment:
Next time please follow this usual syntax:
```
# DONE: pre-migration: Field renamed
```

- `DONE`/`NOTHING TO DO`/`TODO` in upper for indicating  the status
- `pre-migration`/`post-migration`/`end-migration` for indicating where is done
- `<action done>` for explaining a bit what has been done

Reply:
Thanks for the recommendation.